_includes/manuals/2.0/3.1.1_supported_platforms.md

```diff
@@ -16,9 +16,9 @@ The following operating systems are supported by the installer, have packages an
 * Debian 10 (Buster)
   * Architectures: i386, amd64, aarch64
 
-PostgreSQL from Software Collections:
-
-* PostgreSQL version 10
+PostgreSQL version 10 or newer
+  * For Ubuntu and Debian PostgreSQL is available from regular OS repositories.
+  * For CentOS PostgreSQL is available from Software Collections.
```

Comment:
The current text implies you must use them from the OS, but that's not really true. You can also use packages from https://www.postgresql.org/ or other. It really just needs to be 10+. It probably also works with the AWS managed version. I'd limit it to
```suggestion
PostgreSQL version 10 or newer. For EL 7 this is available from Software Collections.
```